src/data/withTableData.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const API = "https://storage.googleapis.com/replaypoker-dummy-api/tables";
+
+export function withTableData(Component) {
```

Comment:
Could you do this as a custom hook? No need to rewrite, just want to know if you are as good at the hook API as with HOCs 🙂

Reply:
Yes, I've been working with hooks on a code refactor for a project that's still early in the development phase and I've written a couple of custom Hooks to interact with the API.